nav.js

```diff
@@ -63,4 +63,83 @@ for (const impl_collapsed of document.getElementsByClassName('impl_collapsed'))
         impl_args[0].addEventListener('click', () =>
             impl_collapsed.classList.remove('impl_collapsed'));
     }
-}
\ No newline at end of file
+}
+
+
+
+function filterSelectionClass(c, classname) {
```

Comment:
```suggestion
function filterSelectionClass(tagNames, classname) {
```

I hope that's what's in `c`.